mintlify/remittances/quickstart.mdx

```diff
@@ -3,4 +3,165 @@ title: "Quickstart"
 ---
 import { umaaasProductName } from '/snippets/variables.mdx'
 
-This guide will walk you through the steps to get started with {umaaasProductName}.
\ No newline at end of file
+This quickstart guides you through sending a payment from the US to a bank account in Mexicousing just‑in‑time funding.
+
+<Steps>
+<Step title="Get API credentials">
+  Create a Sandbox API key and secret in the dashboard, then set environment variables for local use.
+
+  ```bash
+  export GRID_BASE_URL="https://api.lightspark.com/grid/2025-10-13"
+  export GRID_API_KEY="YOUR_SANDBOX_KEY"
+  export GRID_API_SECRET="YOUR_SANDBOX_SECRET"
+  ```
+
+  <Tip>
+  Use Basic Auth in cURL with `-u "$GRID_API_KEY:$GRID_API_SECRET"`.
+  </Tip>
+</Step>
+
+<Step title="Create a customer">
+  Register a customer who will send the payment. You can provide your own UMA handle or let the system generate one.
+
+  ```bash
+  curl -sS -X POST "$GRID_BASE_URL/customers" \
+    -u "$GRID_API_KEY:$GRID_API_SECRET" \
+    -H "Content-Type: application/json" \
+    -d '{
+      "platformCustomerId": "cust_7b3c5a89d2f1e0",
+      "customerType": "INDIVIDUAL",
+      "umaAddress": "$alice@yourapp.example",
+      "fullName": "Alice Smith",
+      "address": {
+        "line1": "123 Pine Street",
+        "city": "Seattle",
+        "state": "WA",
+        "postalCode": "98101",
+        "country": "US"
+      }
+    }'
+  ```
+  
+  Response:
+  ```json
+  {
+    "id": "Customer:019542f5-b3e7-1d02-0000-000000000001",
+    "customerType": "INDIVIDUAL",
+    "umaAddress": "$alice@yourapp.example",
+    "platformCustomerId": "cust_7b3c5a89d2f1e0"
+  }
+  ```
+</Step>
+
+<Step title="Create an external receiving bank account (CLABE in MX)">
+  Add a beneficiary account in Mexico using their CLABE. We attach it to the same customer for this example.
+
+  ```bash
+  curl -sS -X POST "$GRID_BASE_URL/external-accounts" \
+    -u "$GRID_API_KEY:$GRID_API_SECRET" \
+    -H "Content-Type: application/json" \
+    -d '{
+      "currency": "MXN",
+      "platformAccountId": "mx_beneficiary_001",
+      "customerId": "Customer:019542f5-b3e7-1d02-0000-000000000001",
+      "beneficiary": {
+        "counterPartyType": "INDIVIDUAL",
+        "fullName": "Carlos Pérez",
+        "email": "carlos.perez@example.mx"
+      },
+      "accountInfo": {
+        "accountType": "CLABE",
+        "clabeNumber": "123456789012345678",
+        "bankName": "BBVA Mexico",
+        "accountHolderName": "Carlos Pérez"
+      }
+    }'
+  ```
+  
+  Response:
+  ```json
+  {
+    "id": "ExternalAccount:a12dcbd6-dced-4ec4-b756-3c3a9ea3d123",
+    "currency": "MXN",
+    "beneficiary": { "fullName": "Carlos Pérez" },
+    "accountInfo": { "accountType": "CLABE", "bankName": "BBVA Mexico" }
+  }
+  ```
+</Step>
+
+<Step title="Create a quote (just‑in‑time funding)">
```

Comment:
If we're talking about a p2p remittance use case here, you might want to make the lnurl `/receivers/external-accounts/{id}` request to get the rate estimate while a user types the amount to send, etc.